dts/bindings/gpio/gpio-leds.yaml

```diff
@@ -5,6 +5,13 @@ description: GPIO LEDs parent node
 
 compatible: "gpio-leds"
 
+properties:
```

Comment:
This does not belong here either.  If something like this were needed, it'd be called `label`.

Reply:
It can be renamed to `label` or even moved to Kconfig. I would to define here the driver name which can be used for bindings in `device_get_binding`

Reply:
The standard way to do that is:
```
include: base.yaml

properties:
    label:
      required: true
```

Reply:
Agree with @pabigot, this needs to be done with the 'label' property. We have documentation for this here that users rely on:

https://docs.zephyrproject.org/latest/guides/dts/howtos.html#get-a-struct-device-from-a-devicetree-node